package-dev/Plugins/macOS/SentryNativeBridge.m

```diff
@@ -0,0 +1,220 @@
+#import <Foundation/Foundation.h>
+#include <dlfcn.h>
+
+static int loadStatus = -1; // unitialized
+
+static Class SentrySDK;
+static Class SentryScope;
+static Class SentryBreadcrumb;
+static Class SentryUser;
+
+#define LOAD_CLASS_OR_BREAK(name)                                                                  \
+    name = (__bridge Class)dlsym(dylib, "OBJC_CLASS_$_" #name);                                    \
+    if (!name) {                                                                                   \
+        NSLog(@"Sentry (native bridge): Couldn't load %@ class from the dynamic library", name);   \
```

Comment:
what does this print when `name` happens to be null?

Reply:
well spotted, it was supposed to be `#name` (stringified). 

I've changed this code because xcode didn't like the standard constant string concatenation - `"...couldn't load " #name " class..."` - saying it should be a param... When I made it a param I messed it up and used `name` instead of `#name`... thanks